viewers/nuxeo-image-viewer.html

```diff
@@ -164,6 +176,9 @@
               toggleDragModeOnDblclick: false,
               viewMode: 1,
               zoomOnWheel: this.zoomOnWheel,
+              zoom: (data) => {
```

Comment:
I guess you don't need the curly braces? like `zoom: (data) => this._verifyZoomRatio(data),`